keyboards/lz/erghost/matrix.c

```diff
@@ -0,0 +1,442 @@
+/*
```

Comment:
This needs stripping of all unused code. Keeping the COL2ROW and DIRECT_PINS logic just causes maintenance issues.